client/src/shared/Form/AddTelephone.js

```diff
@@ -13,26 +25,91 @@ const AddTelephone = ({ className, formik }) => {
-        defaultCountry="ua"
-        regions="europe"
-        helperText={
-          formik.errors.phone &&
-          formik.touched.phone &&
-          formik.errors.phone
-        }
-        error={formik.errors.phone && formik.touched.phone}
+        render={arrayHelpers => (
+          <div className={classes.paddingTop}>
+            {formik.values.phone &&
+            formik.values.phone.length > 0 ? (
+                formik.values.phone.map((phone, index) => {
+                  const errorMassage = getIn(
+                    formik.errors,
+                    `phone[${index}]`
+                  );
+                  const touched = getIn(
+                    formik.touched,
+                    `phone[${index}]`
+                  );
+                  return (
+                    <div key={generate()}>
+                      <MuiPhoneNumber
+                        name={`phone[${index}]`}
+                        className={className}
+                        value={phone}
+                        onChange={value =>
+                          formik.setFieldValue(
+                            `phone[${index}]`,
+                            value
+                          )
+                        }
+                        onBlur={formik.handleBlur}
+                        defaultCountry="ua"
+                        regions="europe"
+                        helperText={
+                          errorMassage &&
+                        touched &&
+                        errorMassage
+                        }
+                        error={errorMassage && touched}
+                        InputProps={{
+                          endAdornment: (
+                            <InputAdornment
+                              position="end"
+                              onClick={() =>
+                                arrayHelpers.remove(index)
+                              }
+                            >
+                              <IconButton
+                                color="secondary"
+                                aria-label="add to shopping cart"
+                              >
+                                <DeleteIcon fontSize="small" />
+                              </IconButton>
+                            </InputAdornment>
+                          )
+                        }}
+                      />
+                    </div>
+                  );
+                })
+              ) : (
+                <></>
+              )}
+            <div className={classes.centered}>
+              <IconButton
+                color="primary"
+                aria-label="add to shopping cart"
+                onClick={() =>
+                  arrayHelpers.insert(
+                    formik.values.phone.length,
+                    ''
+                  )
+                }
+              >
+                <AddCircleIcon fontSize="large" />
+              </IconButton>
```

Comment:
Show the add new phone button only when a user has fullfilled the previous phone.